### PR TITLE
Rename Kuzzle server to Kuzzle Backend

### PR DESCRIPTION
**Pull request by scottinet**
## What does this PR do ?

Fix https://github.com/kuzzleio/documentation/issues/428: rename "Kuzzle server" to "Kuzzle Backend"

### Boyscout

* the link to our documentation website in the console message was using HTTP instead of HTTPS
* update the `createFirstAdmin` CLI documentation

### CODE REVIEW

File: bin/README.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ Starts a Kuzzle instance in the foreground.
...
 
-You will need it to connect to the admin console
+This is a recommended first step to secure your Kuzzle Backend
 
 ```
 $ kuzzle createFirstAdmin
 ```
 
-will guide you through the creation process of the first admin user and fix the rights to other user types if needed.
+This command will guide you through the creation process of the first admin user and fix the rights to other user types if needed.
```
Comment (etrousset) [inline bin/README.md]: I think we could change this:
...the rights *of* other user types...

File: bin/README.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ Starts a Kuzzle instance in the foreground.
 
 # Create the first administrative user account
 
-You will need it to connect to the admin console
+This is a recommended first step to secure your Kuzzle Backend
```
Comment (etrousset) [inline bin/README.md]: Didn't we decide to call it Kuzzle Server?

File: bin/README.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ You can perform a reset followed by a fixtures and/or mappings import by doing:
 $ kuzzle reset --fixtures /path/to/the/fixtures/file.json --mappings /path/to/the/mappings/file.json
 ```
 
-## Reset the Kuzzle server from a script or cron
+## Reset the Kuzzle Backend from a script or cron
```
Comment (etrousset) [inline bin/README.md]: Same remarque as previously

**Comment by scottinet** [comment]
seems like the issue I was solving is outdated => closing that pr

I'll resubmit a PR with the 2 boyscouts separately

**Pull request closed without merging**